src/Commands/CommercePrintfulCommands.php

```diff
@@ -103,16 +112,53 @@ public function test() {
     try {
       $products = $this->pf->syncProducts();
       if ($products['code'] == 200 && isset($products['result'])) {
```

Comment:
This is not necessary, in such a case a `PrintfulException` will be thrown. If for some reason it'll not happen - we'll update the `Printful` service class so it will be, but so far I haven't experienced such behaviour.

Reply:
Ok, will remove.

Reply:
This should be resolved now...